signature-aggregator/aggregator/aggregator.go

```diff
@@ -121,19 +118,9 @@ func (s *SignatureAggregator) AggregateSignaturesAppRequest(
 		return nil, errNotEnoughConnectedStake
 	}
 
-	// TODO: remove this special handling and replace with ACP-118 interface once available
-	var reqBytes []byte
-	if sourceSubnet == constants.PrimaryNetworkID {
-		req := coreEthMsg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = coreEthMsg.RequestToBytes(coreEthCodec, req)
-	} else {
-		req := msg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = msg.RequestToBytes(codec, req)
-	}
+	reqBytes, err := proto.Marshal(
+		&sdk.SignatureRequest{Message: unsignedMessage.Bytes()},
```

Comment:
We should add justification bytes here as well as the API. In the API they should be hex encoded and since their meaning is VM specific we don't need to do any processing to them either just like with the message. We should only pass the non-zero fields since the ACP-118 spec but in the API layer we should do validation that at least one of `message`, `justification` fields is non-zero. 

I would also rename our API to use `Message` instead of `UnsignedMessage` to match the ACP-118 spec while we are at it

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/408/commits/557262accb902b63364c1ce5e994f08071d45562

Reply:
This isn't done yet. I added the justification everywhere (in the API) _except for_ right here 😅

Reply:
last piece addressed in https://github.com/ava-labs/awm-relayer/pull/408/commits/29cac3003f36112564c3545f2647bbe934609e54